api/src/commonMain/kotlin/Model.kt

```diff
@@ -24,56 +24,56 @@ data class StartPayload(
     val testName: String? = null,
     val isRealtime: Boolean = false,
     val isGlobal: Boolean = false,
-) : JvmSerializable
+)
 
 @Serializable
-data class SessionPayload(val sessionId: String) : JvmSerializable
+data class SessionPayload(val sessionId: String)
 
 @Serializable
-data class SessionDataPayload(val sessionId: String, val data: String) : JvmSerializable
+data class SessionDataPayload(val sessionId: String, val data: String)
 
 @Serializable
 data class CoverPayload(
     val sessionId: String,
     val data: List<EntityProbes>,
-) : JvmSerializable
+)
 
 @Serializable
 class EntityProbes(
     val name: String,
     val test: String = "",
     val probes: List<Boolean>,
-) : JvmSerializable
+)
 
 @Serializable
 data class StopSessionPayload(
     val sessionId: String,
     val testRun: TestRun? = null,
-) : JvmSerializable
+)
 
 @Serializable
 data class AddTestsPayload(
     val sessionId: String,
     val testRun: TestRun? = null,
-) : JvmSerializable
+)
 
 @Serializable
 data class TestRun(
     val name: String = "",
     val startedAt: Long,
     val finishedAt: Long,
     val tests: List<TestInfo> = emptyList(),
-) : JvmSerializable
+)
 
 @Serializable
 data class TestInfo(
     val name: String,
     val result: TestResult,
     val startedAt: Long,
     val finishedAt: Long,
-) : JvmSerializable
+)
 
-enum class TestResult : JvmSerializable {
+enum class TestResult {
```

Comment:
TestResult is part of TestStats which is part of BundleCounters so i guess it should have JvmSerializable interface , does it work with out it ?